server/Arcadia.Assistant.Server.Console/appsettings.json

```diff
@@ -30,26 +30,38 @@
       "ApplicationPushUrls": []
     },
     "CalendarEvents": {
-      "SickLeaveApproved": {
+      "SickLeaveApprovedEmail": {
         "NotificationRecipient": "",
         "NotificationSender": "",
         "Subject": "Approved sick leave",
         "Body": "Dear Sir/Madam,\nNotify you, that employee {0} is on sick leave starting {1}.\nYours faithfully,\nTesting team"
       },
-      "EventAssignedToApprover": {
+      "EventAssignedToApproverEmail": {
         "NotificationSender": "", 
         "Subject": "Pending calendar event",
         "Body": "Dear Sir/Madam,\nNotify you, that {0} event of employee {1} needs your approval.\nYours faithfully,\nTesting team"
       },
-      "EventStatusChanged": {
+      "EventAssignedToApproverPush": {
```

Comment:
please split it on email / push notification sections, easier to read and edit.

Reply:
Done.